### DISCUSSION TITLE
Version mismatch on Github and CRAN

### DISCUSSION THREAD

**Discussion by lucazav** [xiaodaigh/disk.frame#355] [General]
The latest release I can see here on Github is 0.3.0. On [CRAN](https://cran.r-project.org/web/packages/disk.frame/index.html) I can see the 0.5.0 right now.
Why this mismatch? How can I see the changes made to version 0.5.0?

**Comment by xiaodaigh** [comment]
I will move this to discussion since it's not a bug report.

They are out of sync. Let me check how others do it.
